content/develop/sidebars.ts

```diff
@@ -0,0 +1,172 @@
+module.exports = {
+    develop: [
+      {
+        label: 'Smart Contracts',
+        type: 'link',
+        href: '/smart-contracts/overview'
+      },
+      {
+        type: 'category',
+        label: 'Libraries',
+        collapsed: true,
+        items: [
+            {
+                label: 'Client.rs',
+                type: 'link',
+                href: '/iota.rs/welcome',
+            },
+            {
+                label: 'Client.js',
+                type: 'link',
+                href: '/iotajs/welcome',
+            },
+            {
+                label: 'Identity',
+                type: 'link',
+                href: '/identity.rs/introduction',
+            },
+            {
+                label: 'Streams',
+                type: 'link',
+                href: '/streams/welcome',
+            },
+            {
+                label: 'Stronghold',
+                type: 'link',
+                href: '/stronghold.rs/welcome',
+            },
+            {
+                label: 'Wallet.rs',
+                type: 'link',
+                href: '/wallet.rs/welcome',
+            },
+        ],
+      },
+      {
+        type: 'category',
+        label: 'Nodes',
+        collapsed: true,
+        items: [
+            'nodes/about-nodes',
+            'nodes/become-a-node-operator',
+            'nodes/node-software',
+            'nodes/api_reference',
+            {
+              type: 'category',
+              label: 'Explanations',
+              collapsed: false,
+              items: [
+                  'nodes/explanations/nodes_101',
+                  'nodes/explanations/security_101',
+              ]
+            },
+        ],
+      },
+      {
+        type: 'category',
+        label: 'Node Software',
+        collapsed: true,
+        items: [
+            {
+                label: 'Hornet',
+                type: 'link',
+                href: '/hornet/welcome',
+            },
+            {
+                label: 'Bee',
+                type: 'link',
+                href: '/bee/welcome',
+            },
+        ],
+      },
```

Comment:
Perhaps we can combine these 2? Maybe even add INX plugins as a subcategory

Reply:
I just followed the miro board. I actually like the current structure better :trollface: 
Maybe something like:
```
Node Software/
├── Hornet
├── Bee
├── INX Plugins/
│   ├── Dashboard
│   ├── Spammer
│   ├── Poi
│   └── ...
└── Extensions/
    ├── Wasp
    └── Chronicle
```

We could also add SC and Permanode to the last two. But we at least need some introduction here somewhere. Nobody knows what Wasp and Chronicle ist 😅